ocw/lib/azure.py

```diff
@@ -100,126 +99,44 @@ def list_disks_by_resource_group(self, resource_group):
 
     def list_by_resource_group(self, resource_group, filters=None):
         return [item for item in self.resource_mgmt_client().resources.list_by_resource_group(
-            resource_group, filter=filters)]
-
-    def get_keeping_image_names(self):
-        images = list()
-        for item in self.container_client('sle-images').list_blobs():
-            m = self.parse_image_name(item.name)
-            if m:
-                images.append(Image(item.name, flavor=m['key'], build=m['build'], date=item.last_modified))
-            else:
-                self.log_err("Unable to parse image name '{}'", item.name)
-
-        return super().get_keeping_image_names(images)
+            resource_group, filter=filters, expand="changedTime")]
 
     def cleanup_all(self):
-        ''' Cleanup all autodateed data which might created during automated tests.'''
-        self.cleanup_bootdiagnostics()
-
-        keep_images = self.get_keeping_image_names()
-        self.cleanup_sle_images_container(keep_images)
-        self.cleanup_disks_from_rg(keep_images)
-        self.cleanup_images_from_rg(keep_images)
-        for i in keep_images:
-            self.log_info("Keep image {} ", i)
-
-    def cleanup_bootdiagnostics(self):
-        containers = self.bs_client().list_containers()
+        self.cleanup_blob_containers()
```

Comment:
I think blobs need to be deleted at last.

Reply:
done